roles/swift-common/tasks/main.yml

```diff
@@ -76,6 +76,11 @@
     - logging
   when: logging.enabled|default('True')|bool
 
+- name: change swift log file permissions
+  file: path={{ item }} mode=0644 owner=swift group=swift state=touch
```

Comment:
we should discuss a better approach on slack Tuesday. I agree, we wouldnt want to break syslog.